packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -612,12 +618,13 @@ class DragTarget<T extends Object> extends StatefulWidget {
     super.key,
     required this.builder,
     this.onWillAccept,
+    this.onWillAcceptWithDetails,
```

Comment:
Is this meant to replace `onWillAccept`? Should `onWillAccept` be deprecated in favor of this new API?

Reply:
Based on what is highlighted by @goderbauer [here](https://github.com/flutter/flutter/issues/55184#issuecomment-616700139), `onWillAceeptWithDetails` would eventually replace `onWillAccept` completely.

Reply:
Sounds good, when we run through triage today I will sync with @goderbauer on this. Before we land it, we should have a plan for deprecation and migration.

Reply:
Which could be relatively simple for this type of change, dart fix can rename this for folks really easily. :)